### ISSUE TITLE
Login prompt for my username and password and does mfa but then nothing similar to issue #13

### ISSUE THREAD

**Issue by rsnyder-feld** [OfficeDev/microsoft-teams-emergency-operations-center#49]
I have now been though the deployment guide 3 times and it does the same thing in teams on my pc and on my mac 

The teams apps loads 
![image](https://user-images.githubusercontent.com/115168361/194342039-48aca2af-ed8e-405e-9537-0927e5265720.png)

I press login it prompts me to sign into AAD ( or on windows it just auto does it ) it prompts for mfa i put in my azure authenicator code . then nothing. 

I checked my manifest file 
![image](https://user-images.githubusercontent.com/115168361/194342531-d37d1dc7-552e-4aec-9690-aba594d309fd.png)

Not sure where to look next

if it tell it to go to site in teams I just get a never ending please wait but did not have any issues in the deployment 
![image](https://user-images.githubusercontent.com/115168361/194342925-273d2685-ea3e-4725-b1a4-712c0cd1ce74.png)
![image](https://user-images.githubusercontent.com/115168361/194343230-5f246559-bcde-4a20-916b-ef4f5864191e.png)

Any guidance would be helpful and appreciated

**Comment by rsnyder-feld** [comment]
[consolelogs.txt](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/files/9726203/consolelogs.txt)

I never tried ( teams in browser ) but in a browser it is working but not in the app.

![image](https://user-images.githubusercontent.com/115168361/194352766-8000fd31-fec9-4adc-bfcb-77d1c022ac1b.png)

**Comment by rsnyder-feld** [comment]
This may be an issue of personal stupidity. 

I had made the teams app package and I had a copied the wrong client ID and uploaded it into my org app catalog. 

I realized the mistake and delete it from the teams admin center, fixed the client ID and re uploaded but I was getting the same  login page bs.  

now it working in the Teams app as well. it looks I did now wait long enough to let everything clear out.
